api/specs/webserver/v0/components/schemas/project-v0.0.1.json

```diff
@@ -97,20 +72,19 @@
           "type": "object",
           "additionalProperties": false,
           "required": [
+            "key",
+            "version",
             "label",
             "position"
           ],
           "properties": {
-            "label": {
-              "type": "string"
-            },
             "key": {
               "type": "string",
               "description": "distinctive name for the node based on the docker registry path",
-              "pattern": "^(simcore)/(services)(/demodec)?/(comp|dynamic)(/[^\\s/]+)+$",
```

Comment:
can't we have a pattern that works for the registry key and at the same time for the others? now it's fully free... can we have either simcore/service/dyn|comp or a name such as group, file-picker?

Reply:
Resolved in https://github.com/pcrespov/osparc-simcore/pull/21

Reply:
and I guess /demodec should be removed?